apps/general/models.py

```diff
@@ -1,3 +1,230 @@
+    iec_code = models.CharField(max_length=50, default=None, null=True)
+    eway_username = models.CharField(max_length=100, default=None, null=True)
+    eway_password = models.CharField(max_length=100, default=None, null=True)
+    gstn_username = models.CharField(max_length=100, default=None, null=True)
+    gstn_password = models.CharField(max_length=100, default=None, null=True)
+    VAT_GST_STATUS_CHOICES = (
+        ('Active', 'Active'),
+        ('Inactive', 'Inactive'),
+        ('Pending', 'Pending'),
+    )
+    vat_gst_status = models.CharField(max_length=10, choices=VAT_GST_STATUS_CHOICES, default=None, null=True)
+    GST_TYPE_CHOICES = (
+        ('Goods', 'Goods'),
+        ('Service', 'Service'),
+    )
+    gst_type = models.CharField(max_length=10, choices=GST_TYPE_CHOICES, default=None, null=True)
+    einvoice_approved_only = models.BooleanField(default=False)
+    marketplace_url = models.URLField(max_length=255, default=None, null=True)
+    drug_license_no = models.CharField(max_length=50, default=None, null=True)
+    other_license_1 = models.CharField(max_length=50, default=None, null=True)
+    other_license_2 = models.CharField(max_length=50, default=None, null=True)
+    turnover_less_than_5cr = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    is_deleted = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f"{self.company_id}.{self.name}"
+
+    class Meta:
+        db_table = 'companies'
+
+    def save(self, *args, **kwargs):
+        if self.eway_password:
+            #Here I am Hashig the eway_password using bcrypt and save as bytes
+            hashed_eway_password = bcrypt.hashpw(self.eway_password.encode(), bcrypt.gensalt())
+            self.eway_password = hashed_eway_password
+        if self.gstn_password:
+            #Here I am Hashig the gstn_password using bcrypt and save as bytes
+            hashed_gstn_password = bcrypt.hashpw(self.gstn_password.encode(), bcrypt.gensalt())
+            self.gstn_password = hashed_gstn_password
+        super().save(*args, **kwargs)
+
+    def verify_eway_password(self, password):
+        #Here I am Verifying the eway_password using bcrypt
+        if self.eway_password:
+            return bcrypt.checkpw(password.encode(), self.eway_password)
+        return False
+
+    def verify_gstn_password(self, password):
+        #Here I am Verifying the gstn_password using bcrypt
+        if self.gstn_password:
+            return bcrypt.checkpw(password.encode(), self.gstn_password)
+        return False
+
+def branches_picture(instance, filename):
+    # Assuming 'instance' is an instance of our model
+    branch_name = instance.name.replace(' ', '_')
+    unique_id = uuid.uuid4().hex[:6]  # Generate a unique identifier
+    return f"branches_pictures/{branch_name}_{unique_id}/{filename}"
```

Comment:
THis picture should store in media folder, did you verify?

Reply:
Yeah Thulasi I forgot to give the media path in settings. So that is why it is not storing I will add that path in settings then it will store automatically in the folder.